components/org.wso2.carbon.identity.oauth.endpoint/src/test/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpointTest.java

```diff
@@ -2498,4 +2499,25 @@ public void testHandleOAuthProblemException(String error, String description, St
         assertEquals(location, expectedUrl);
     }
 
+    @Test
+    public void testPKCEValidationForDeviceCodeGrantFlow() throws Exception {
```

Comment:
Make the test generic. no need to be specific for the device code grant

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2112/commits/74e3d3d8e1bc627ff897c62fc3ca24cf1b154663